getting-started/concepts.mdx

```diff
@@ -6,54 +6,54 @@ icon: 'building-columns'
 
 ## Notification
 
-A notification conveys information from source to recipient, triggered by a workflow acting as a message blueprint. Notifications can be individual or bundled as digest for user-friendliness.
+A notification conveys information from source to recipient, triggered by a workflow acting as a message blueprint. Notifications can be individual or bundled as digest for user-friendliness. 
```

Comment:
Link the digest word in this sentence to a part of the docs that explains digest in full.

Reply:
Done.